gammapy/maps/regionnd.py

```diff
@@ -154,6 +155,37 @@ def plot_region(self, ax=None, **kwargs):
         ax = self.geom.plot_region(ax, **kwargs)
         return ax
 
+    def plot_mask(self, ax=None, **kwargs):
+        """Plot the mask as a shaded area in a xmin-xmax range
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axis` 
+            Axis instance to be used for the plot.
+        **kwargs : dict
+            Keyword arguments passed to `~matplotlib.pyplot.axvspan`
+        """
+        import matplotlib.pyplot as plt
+
```

Comment:
Include a check like `if not self.is_mask:` and raise an error to forbid the plotting of non-masks

Reply:
Ok.